MethodsAndParameterizedTests/src/TestMethods.java

```diff
@@ -0,0 +1,66 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+public class TestMethods {
+
+    @ParameterizedTest
+    @ValueSource(strings = {"kutya", "macsek", "hulLa", "tundra", "huH", "ciCa", "oltár"})
+    void shouldReturnTrueToAreUniqueCharacters(String string) {
+        WordsAndChars wac = new WordsAndChars();
+        Assertions.assertTrue(wac.areUniqueCharacters(string));
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"alma", "hulla", "papa", "anya", "macska", "olló", "CiCa"})
+    void shouldReturnFalseToAreUniqueCharacters(String string) {
+        WordsAndChars wac = new WordsAndChars();
+        Assertions.assertFalse(wac.areUniqueCharacters(string));
+    }
+
+    @ParameterizedTest
+    @MethodSource("provideSameStringInDifferentCombination")
+    void shouldReturnCountedWords(List<String> strings) {
+        WordsAndChars wac = new WordsAndChars();
+        Map<String, Integer> countedWords = new HashMap<>();
+        countedWords.put("macska", 2);
+        countedWords.put("tehén", 2);
+        countedWords.put("kutya", 1);
+
+        Map<String, Integer> result = wac.countWords(strings.get(0));
+
+        Assertions.assertEquals(countedWords, result);
+    }
+
+    static Stream<List<String>> provideSameStringInDifferentCombination() {
```

Comment:
Love your method name! One minor thing: I would rename it to `provideSameStringInDifferentCombinations`, since you are returning multiple different combinations here